doc.html

```diff
@@ -1012,5 +1012,8 @@
                 $(".searchBtn").show();
             }
         }
+
+        // add margin between body content and footer to avoid overlap
+        $('.section_card').css('margin-bottom', $('.footer').height());
```

Comment:
Correct me if I am wrong and I don't really have a good understanding here but I have a naive question. So you added margin-bottom that is equal to the height of the footer but what if the document text content is bigger than that of the footer height would that result in overlap again?

Reply:
So current situation is exactly the same as you have asked in the question.
Footer is a sticky footer( [css](https://developer.mozilla.org/en-US/docs/Web/CSS/position) property) it won't move even if you scroll and document text(last few lines) is getting overlapped due to this behavior.

What we need is doc text should be display before the footer starts, so i have added margin-bottom (wide space) equal to footer height(this will add wide space after the document text ends). Also in future if footer content increases and result in increasing the footer size(height), this fix would adapt and adjust the rendering.